<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsDiag</class>
 <widget class="QDialog" name="ResultsDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="resultTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>531</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="previewEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultsLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Results:</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultNumLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyAllBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy All</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="preSpin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>389</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="postSpin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>389</y>
     <width>48</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="resultsLabel_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>pre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultsLabel_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>post:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="contextChk">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>391</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy Context</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>resultsLabel_4</zorder>
  <zorder>resultsLabel_3</zorder>
  <zorder>resultTable</zorder>
  <zorder>previewEdit</zorder>
  <zorder>selectBtn</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>resultsLabel</zorder>
  <zorder>resultNumLabel</zorder>
  <zorder>copyAllBtn</zorder>
  <zorder>preSpin</zorder>
  <zorder>postSpin</zorder>
  <zorder>contextChk</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
